core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -196,6 +196,7 @@ registry.ci.openshift.org/origin/4.10:openstack-cinder-csi-driver-operator quay.
 registry.ci.openshift.org/origin/4.10:openstack-cinder-csi-driver-operator-test quay.io/openshift/origin-openstack-cinder-csi-driver-operator-test:4.10 quay.io/openshift/origin-openstack-cinder-csi-driver-operator-test:4.10.0 quay.io/openshift/origin-openstack-cinder-csi-driver-operator-test:latest
 registry.ci.openshift.org/origin/4.10:openstack-cloud-controller-manager quay.io/openshift/origin-openstack-cloud-controller-manager:4.10 quay.io/openshift/origin-openstack-cloud-controller-manager:4.10.0 quay.io/openshift/origin-openstack-cloud-controller-manager:latest
 registry.ci.openshift.org/origin/4.10:openstack-machine-controllers quay.io/openshift/origin-openstack-machine-controllers:4.10 quay.io/openshift/origin-openstack-machine-controllers:4.10.0 quay.io/openshift/origin-openstack-machine-controllers:latest
+registry.ci.openshift.org/origin/4.10:openstack-tester quay.io/openshift/origin-openstack-tester:4.10 quay.io/openshift/origin-openstack-tester:4.10.0 quay.io/openshift/origin-openstack-tester:latest
```

Comment:
This image contains a binary called `openstack-test`. Its code sits in https://github.com/openshift/openstack-test .

I didn't call the image `openstack-test` because in the past I created a `config` where an image had the same name as a `test`, and the former overwrote the latter in ci-operator.

In other words: you can't have an image and a test called the same _in one config_.

I am happy renaming the image to `openstack-test` if this won't cause issues when we call a step, or a test stanza, or whatever else `openstack-test`.

Reply:
@smarterclayton 
I am not clear if based on my feedback, ‘openstack-test’ is still the name you want. I will change to whatever you see fit

Reply:
Ping @smarterclayton ?